### PR TITLE
Sources for all rules in single and batch mode

### PR DESCRIPTION
**Pull request by tbedford**
## Description

The four rule sources are now supported for all rules, in both single and batched mode.

* [DOC-477](https://ably.atlassian.net/browse/DOC-477)

## Checklist for OpenAPI document updates

Please ensure the following:

- [x] Version has been incremented
- [x] Tested for errors with Spectral
- [x] Optional: Load into SwaggerHub and check for errors
- [x] Ensure the document renders under ReDoc (see README for instructions on how to test locally)

## Review

* Check the diff
* Test with Spectral
* Check version number has been incremented
* Check all renders correctly

### CODE REVIEW

**Comment by tbedford** [comment]
#20 should be merged first.

File: definitions/control-v1.yaml [hunk 1]
```diff
@@ -1187,7 +1187,7 @@ components:
...
+          description: "Ably currently supports the following sources for all rule types, in both single and batch mode: `channel.message`, `channel.presence`, `channel.lifecycle`, `channel.occupancy`. If the source `channel.message` is selected, messages trigger function calls as soon as they are published on a channel. If the source `channel.presence` is selected, presence events trigger function calls when clients enter, update their data, or leave channels. If the source `channel.lifecycle` is selected, you receive notifications for channel lifecycle events, such as when a channel is created (following the first client attaching to this channel) or discarded (when there are no more clients attached to the channel). If the source `channel.occupancy` is selected, you receive notifications for occupancy events, which relate to the number and type of occupants in the channel."
```
Comment (m-hulbert) [inline definitions/control-v1.yaml:1190]:
```suggestion
          description: "Ably currently supports the following sources for all rule types, in both single and batch mode: `channel.message`, `channel.presence`, `channel.lifecycle` and `channel.occupancy`. If the source `channel.message` is selected, messages trigger function calls as soon as they are published on a channel. If the source `channel.presence` is selected, presence events trigger function calls when clients enter, update their data, or leave channels. If the source `channel.lifecycle` is selected, you receive notifications for channel lifecycle events, such as when a channel is created (following the first client attaching to this channel) or discarded (when there are no more clients attached to the channel). If the source `channel.occupancy` is selected, you receive notifications for occupancy events, which relate to the number and type of occupants in the channel."
```

File: definitions/control-v1.yaml [hunk 2]
```diff
@@ -1202,7 +1202,7 @@ components:
...
```
Comment (m-hulbert) [inline definitions/control-v1.yaml:1205]:
```suggestion
          description: "Ably currently supports the following sources for all rule types, in both single and batch mode: `channel.message`, `channel.presence`, `channel.lifecycle` and `channel.occupancy`. If the source `channel.message` is selected, messages trigger function calls as soon as they are published on a channel. If the source `channel.presence` is selected, presence events trigger function calls when clients enter, update their data, or leave channels. If the source `channel.lifecycle` is selected, you receive notifications for channel lifecycle events, such as when a channel is created (following the first client attaching to this channel) or discarded (when there are no more clients attached to the channel). If the source `channel.occupancy` is selected, you receive notifications for occupancy events, which relate to the number and type of occupants in the channel."
```

**Review by m-hulbert — CHANGES_REQUESTED** [review]
This builds with no problems and the version has been incremented.

My only other suggestion would be that `channel.message` and `channel.presence` read differently to the other two sources - they refer to the events triggering the function calls whereas `channel.lifecycle` and `channel.occupancy` refer to "you receiving notifications..." I'd suggest we standardise on one or the other.

**Review by m-hulbert — APPROVED** [review]
LGTM

**Pull request merged**